pkg/operator/operator.go

```diff
@@ -32,8 +36,23 @@ type Operator struct {
 }
 
 func NewOperator(ctx context.Context, operator *operator.Operator) (context.Context, *Operator) {
+	aliRegion := region.GetRegion()
+
+	ecsClient, err := client.NewECSClient(aliRegion)
+	if err != nil {
+		log.FromContext(ctx).Error(err, "failed to create ecs client")
```

Comment:
What do u think? I prefer klog, it's more clear for me

Reply:
I prefer to keep it consistent with karpenter core. 
`log.FromContext(ctx)`